backend/apps/github/models/repository.py

```diff
@@ -106,6 +107,7 @@ def is_indexable(self):
             not self.is_archived
             and not self.is_empty
             and not self.is_template
+            and self.sync_issues
```

Comment:
```suggestion
            and self.sync_issues
```
You need to modify issues indexing condition instead.

Reply:
How? Isn't it already modified?

Reply:
Please see the implementation in the recent commit.